promote/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian as final
+RUN apt-get update && apt-get install -y curl unzip git ssh
```

Comment:
You should clear the apt cache when you're done to improve caching of the docker image